vispy/visuals/volume.py

```diff
@@ -120,16 +120,10 @@
 }}
 
 vec4 applyColormap(float data) {{
-    if (clim.x < clim.y) {{
-        data = clamp(data, clim.x, clim.y);
-    }} else if (clim.x > clim.y) {{
-        data = clamp(data, clim.y, clim.x);
-    }} else {{
-        return $cmap(0.0);
-    }}
-
     data = (data - clim.x) / (clim.y - clim.x);
-    return $cmap(pow(data, gamma));
+    data = clamp(data, 0.0, 1.0);
```

Comment:
It may be too early for me to be reviewing this code again, but is there any worry about data type and overflow with doing this clamp after the scaling? Like data that's outside the clim range?

Reply:
I wondered about this too, but since it's all floats, I don't think it should matter (e.g. no int overflow).

Reply:
mmm, could put it back up and do `clamp(data, min(clim.x, clim.y), max(clim.x, clim.y))` though, just in case ...

Reply:
done